torchci/README.md

```diff
@@ -85,3 +85,13 @@ console.
 2. Save the query, creating a new version.
 3. Download the query with `yarn node scripts/downloadQueryLambda.mjs <workspace> <queryname> <version>`.
 4. Update `rockset/prodVersion.json` with the new version of the lambda.
+
+
+### Modifying Deployment Settings
+
+If you ever need to modify the deployment settings like the oauth callbacks, domain names, there's a few places that you need to change these settings in. Here's a list:
+
+1. DNS Registry/Certificates (Owned by the OSS team)
```

Comment:
So is the suggestion here to contact the OSS team? If so, we should suggest a PoC or link to the group of Open Source Support.